modules/phase_field/include/postprocessors/FauxGrainTracker.h

```diff
@@ -44,14 +41,14 @@ class FauxGrainTracker : public FeatureFloodCount, public GrainTrackerInterface
    * @param show_var_coloring pass true to view variable index for a region, false for unique grain information
    * @return the entity value
    */
-  virtual Real getEntityValue(dof_id_type entity_id, FeatureFloodCount::FIELD_TYPE field_type, unsigned int var_idx) const;
+  virtual Real getEntityValue(dof_id_type entity_id, FeatureFloodCount::FIELD_TYPE field_type, unsigned int var_idx) const override;
```

Comment:
Yes as we start committing new code. I don't think we necessarily need to sweep through everything but it has a lot of benefits. I think one of my favorite benefits is that it indicates that you probably don't need to comment on the overridden method.